### PR TITLE
Added a lot of extensions and filenames, please have a look

### PR DESCRIPTION
**Pull request by Babkock**
Extensions I've added:

- **.a** - Library file for Unix
- **.asm** - Assembly source file
- **.aup** - Audacity project
- **.class** - Java class file
- **.cmake** - CMake directives
- **.dll** - Dynamic linked library for Windows
- **.doc** - Microsoft Word document
- **.docx** - Microsoft Word document
- **.efi** - EFI boot loader
- **.elf** - Executable and Linkable Format
- **.fifo** - Named pipe
- **.h** - C/C++ header file
- **.htaccess** - Apache directives
- **.htpasswd** - Apache password hash
- **.img** - Disk image / Initramfs
- **.iso** - Disk image
- **.jar** - Java file
- **.key** - Domain name key for SSH/SSL
- **.m4a** - Apple Lossless audio
- **.m4v** - Apple MP4 video
- **.msi** - Microsoft installer
- **.o** - Object file
- **.ppt** - Microsoft Powerpoint document
- **.pub** - SSH public key file
- **.rom** - Read-Only Memory image
- **.s** - Assembly source file
- **.so** - Shared object library for Unix
- **.xbps** - Void Linux binary package
- **.yaml** - YAML

Filenames:

- **a.out** - Assembler output
- **.bash_aliases**
- **.bash_logout**
- **.fehbg**
- **.inputrc**
- **.viminfo**
- **.xinitrc** - X startup script
- **Makefile**
- **Rakefile** - Ruby Rake Makefile
- **config.mk** - Make directives
- **known_hosts** - Known hosts for SSH
- **authorized_keys** - SSH
- **LICENSE** and derivatives
- **README** and derivatives
- **webpack.config.js** - Webpack config file
- **package.json** - NPM package config

### CODE REVIEW

File: devicons.py [hunk 1]
```diff
@@ -138,6 +172,7 @@
     'Desktop'                          : '',
     'Documents'                        : '',
     'Downloads'                        : '',
+    'Dotfiles'                         : '',
```
Comment (alexanderjeurissen) [inline devicons.py]: Instead of this being the same icon as dropbox, I think having it be the `cog` icon makes more sense.

**Comment by alexanderjeurissen** [comment]
Thanks for this great contribution, I have one small comment curious what your thinking is.

File: devicons.py (hunk 1 above)
Comment (Babkock) [reply devicons.py]: Ah yes, I agree. I went ahead and changed it.

**Comment by alexanderjeurissen** [comment]
Awesome work, I'll go ahead and merge it. 

Thanks again for your contribution

**Pull request merged**